--- StatusArea-old.qml	2020-08-13 21:28:56.000000000 +0100
+++ StatusArea.qml	2020-08-13 22:00:54.387486006 +0100
@@ -111,17 +111,11 @@
         Component {
             id: timeText
             ClockItem {
-                id: clock
 
                 width: Math.min(implicitWidth, centralArea.width)
                 updatesEnabled: recentlyOnDisplay
                 color: Theme.primaryColor
                 font { pixelSize: Theme.fontSizeMedium; family: Theme.fontFamilyHeading }
-
-                Connections {
-                    target: Lipstick.compositor
-                    onDisplayAboutToBeOn: clock.forceUpdate()
-                }
             }
         }
         Component {
@@ -142,6 +136,22 @@
                 right: parent.right
                 rightMargin: Theme.paddingMedium
             }
+
+            ClockItem {
+                id: clock
+
+                width: Math.min(implicitWidth, centralArea.width)
+                y: (parent.height - height)/2
+                updatesEnabled: recentlyOnDisplay
+                color: Theme.primaryColor
+                font { pixelSize: Theme.fontSizeMedium; family: Theme.fontFamilyHeading }
+
+                Connections {
+                    target: Lipstick.compositor
+                    onDisplayAboutToBeOn: clock.forceUpdate()
+                }
+            }
+
             VpnStatusIndicator {
                 id: vpnStatusIndicator
                 anchors.verticalCenter: parent.verticalCenter
